py_hw2/task_3.py

```diff
@@ -0,0 +1,16 @@
+# 3 Задайте список из n чисел последовательности (1 + 1 / n(текущее значение в цикле)) ** n
+#  и выведите на экран их сумму.
+# Пример:
+# 1 -> 2.0
+# 2 -> 4.25
+# 3 -> 6.62037037037037
+
+
+n = int(input('Введите число n: '))
+lst = []
+sum = 0
+for i in range(1, n + 1):
+    num = (1 + 1 / i) ** i
+    sum = sum + num
+    lst.append(sum)
+print(lst)
```

Comment:
```suggestion
    num = (1 + 1 / i) ** i
    sum += num
    lst.append(num)
print(num)
```
Нужно было вывести сумму, а не список сумм.